tests/unit/test_client.py

```diff
@@ -56,27 +58,27 @@ def test_client_search(client, httpx_mock, auth_token):
         )
     )
     assert len(res) == 2
-    assert res[0].id == 1
-    assert res[1].id == 2
+    assert res[0].id == MOCK_UUID
+    assert res[1].id == MOCK_UUID
```

Comment:
Not specific to this line, but I'm a bit concerned that by always using the same MOCK_UUID we can miss some trivial bugs like swapping the values by mistake, what do you think?

Reply:
Indeed. I improved the client tests with varied uuids.